src/cli.js

```diff
@@ -38,6 +40,7 @@ export default class Main {
   async run () {
     const updater = new Updater(out)
     const plugins = new Plugins(out)
+    await MigrateV5Plugins.run([], this.config)
```

Comment:
seems ok, I did that before a lot but then moved away from it, but now things have been refactored a lot it might be ok